### PR TITLE
Revert "Revert "Direct dispatch by default when group size is 1""

### PR DESCRIPTION
**Pull request by bratseth**
Reverts vespa-engine/vespa#5539

### CODE REVIEW

File: container-search/src/main/java/com/yahoo/search/handler/SearchHandler.java [hunk 1]
```diff
@@ -117,6 +120,18 @@ public void run(Handle h, boolean firstTime) {
         }
     }
 
+    private void warmupQueries() {
+        String[] requestUris = new String[] {"warmupRequestUrl1", "warmupRequestUrl2"};
```
Comment (baldersheim) [inline container-search/src/main/java/com/yahoo/search/handler/SearchHandler.java]: This was probably not intended in this pull request ?

**Review by baldersheim — CHANGES_REQUESTED** [review]
Generally loog fine, but pull request build says no.

File: container-search/src/main/java/com/yahoo/search/handler/SearchHandler.java (hunk 1 above)
Comment (bratseth) [reply container-search/src/main/java/com/yahoo/search/handler/SearchHandler.java]: Thanks - removed

**Review by baldersheim — COMMENTED** [review]
:+

**Pull request merged**